src/test/service-worker_test.ts

```diff
@@ -185,6 +185,34 @@ suite('service-worker', () => {
           });
     });
 
+    test('basePath should prefix resources', () => {
+      return serviceWorker
+          .generateServiceWorker({
+            project: defaultProject,
+            buildRoot: testBuildRoot,
+            basePath: '/my/base/path'
+          })
+          .then((swFile: Buffer) => {
+            const fileContents = swFile.toString();
+            assert.include(fileContents, '"/my/base/path/index.html"');
+          });
+    });
+
+    test('basePath prefixes should not have double delimiters', () => {
```

Comment:
But I see you were just following the pre-existing paradigm in the tests, so that's not a dealbreaker on this PR.

Add a note to top of file?